src/components/Swipeable.tsx

```diff
@@ -44,7 +44,9 @@ interface SwipeableProps
   onSwipeableLeftOpen?: () => void;
   onSwipeableRightOpen?: () => void;
   onSwipeableOpen?: () => void;
-  onSwipeableClose?: () => void;
+  // onSwipeableClose?: () => void;
+  // Changes were made to know how much swipe occurred and invoke action based on swipe distance.
+  onSwipeableClose?: (x: Animated.AnimatedInterpolation) => void;
```

Comment:
Why it has to be `Animated.AnimatedInterpolation`? What if the user doesn't use Animated at all?